contributors/contributors/social-engagement-guidelines.md

```diff
@@ -0,0 +1,150 @@
+# Social Engagement Guidelines
+
+This playbook outlines the processes and practices used to engage with the Mattermost community across social community channels.
+
+## Goal
+
+Improve the Mattermost brand by responding to all Mattermost mentions and questions asked online in a timely manner.
+
+## Principles
+
+1. **Earn Trust** - Take actions that maximize the trust of others. Be empathetic, consistent, knowledgeable and truthful.
+
+## General Guidelines
+
+*Adopted from https://handbook.mattermost.com/contributors/contributors/community-playbook#response-writing-tips*
+
+Ultimately, when responding to users you should always keep the Golden Rule in mind -- if you were the person on the other end, how would you want someone to respond to you?
+
+1. **Be professional**
+
+* Maintain professionalism in your responses, as you are speaking as a member of the company. Avoid slang or colloquialisms that may not translate well across other languages.
+* Do not use humour in your responses, as it can be received in the wrong way by the community. Instead, stay factual when answering questions.
```

Comment:
```suggestion
* Do not use humor in your responses, as it can be received in the wrong way by the community. Instead, stay factual when answering questions.
```